compiler/options.nim

```diff
@@ -546,6 +546,9 @@ proc disableNimblePath*(conf: ConfigRef) =
   incl conf.globalOptions, optNoNimblePath
   conf.lazyPaths.setLen(0)
```

Comment:
Should the order of `disableNimblePath` and `clearNimblePath` be switched and have `disableNimblePath` call the other to do that `setLen(0)`?